server/routes/hooks.js

```diff
@@ -13,7 +14,14 @@ export default () => {
 
   hooks.delete('/on-uninstall', (req, res) => {
     logger.debug('Uninstall running...');
-    req.auth0.clients.delete({ client_id: config('AUTH0_CLIENT_ID') })
+    const clientId = config('AUTH0_CLIENT_ID');
```

Comment:
Middleware may return error (`next(err)`), if client was removed before uninstalling extension, for instance. And dashboard requires successful response from hook to proceed with uninstall.
I would suggest to add another confirmation dialog to the dashboard to show customer "uninstall error" and ask him what to do next (retry hook / delete extension despite error / cancel). But so far, the only way to uninstall extension with broken hook is using wt-cli.

Reply:
I would keep it as it is implemented right now. If the `client` is removed by the customer, there will be an error message in the Webtask log, but the extension is successfully removed. On the other hand, if it fails to delete because of another reason and the client isn't removed it will be there for nothing, but can be easily removed via the dashboard.

@mostekcm WDYT?